### DISCUSSION TITLE
Move instead of sync?

### DISCUSSION THREAD

**Discussion by respencer** [floccusaddon/floccus#1568] [General]
Apologies, I don't know what to name what I need or how to do it or if it is even possible.

It's company policy not to store any data on a work computer. I like the idea -- reinforced by stolen mobile phones (whoops -- that backup wasn't as reliable as I thought it was).

I don't want to lose any info and I don't want it hanging around on any of my devices.

Is there any way to automatically move bookmarks in Firefox out of the browser (except for those in the bookmark toolbar) and to a location off of the device?

**Comment by marcelklehr** [comment]
Hi @respencer 

You could do this manually, by disabling auto-sync in floccus then when you want the bookmarks gone, simply delete them. When you need them again, sync down; when you've updated them sync up.
